README.md

```diff
@@ -5,19 +5,69 @@
 PyOpenWorm
 ===========
 
-A unified, simple data access library in Python for data, facts, and models of
-*C. elegans* anatomy for the [OpenWorm project](http://www.openworm.org)
+A data access layer in Python which integrates disparate structures
+and representations for *C. elegans* anatomy and physiology.  Provides a
+clean, high-level interface used by the rest of
+[OpenWorm](http://www.openworm.org).
+
+Overview
+--------
+The data and models required to simulate *C. elegans* are highly heterogenous.
+Consequently, from a software perspective, a variety of underlying
+representations are needed to store different aspects
+of the relevant anatomy and physiology.  For example,
+a NetworkX representation of the connectome as a complex graph enables
+questions to be asked about nearest neighbors of a given neuron.
+An RDF semantic graph representation is useful for reading and
+writing annotations about multiple aspects of a neuron, such as what papers
+have been written about it, properties it may have such as
+ion channels and neurotransmitter receptors, etc.  A NeuroML representation is useful
+for answering questions about model morphology and simulation parameters.  A
+Blender representation is a full 3D shape definition that can be used for
+calculations in 3D space.
+
+The diversity of underlying representations required for OpenWorm
+presents a challenge for data integration and consolidation.  PyOpenWorm solves
+this challenge with a unified data access layer whereby different representations
+are encapsulated into an abstract view.  This allows the user to work with
+objects related to the *biological reality of the worm*, and
+forget about which representation is being used under the hood.  The worm
+itself has a unified sense of neurons, networks, muscles,
+ion channels, etc. and so should our code.
+
+Syntactical constructs in PyOpenWorm reflect the structure of
+the corresponding biological knowledge-base
+rather than the programming idioms of the underlying technologies. Wherever possible,
+pure Python libraries or those with few compilation requirements are used.
+
+Relationship to ChannelWorm
```

Comment:
We talked about adding this here before, but now I'm thinking it doesn't belong here, but in ChannelWorm, if we're going to have the statement. It doesn't make sense to me for a middleware to explain that it's relationship to an application that happens to use it.

Reply:
My specific goal in adding this paragraph was to communicate exactly that point- that it is middleware and not the end application itself.  But because PyOW happens to have a considerable amount of physiological data at present, this fairly important issue is not entirely obvious.  I personally didn't understand it for a long time.  Happy to take this out, but just wanted to make clear why I put that in to begin with.

Reply:
I see the tension, but I don't see a *special* ambiguity between PyOW and ChannelWorm that wouldn't exist between PyOW and any other application that uses it. That said, it may be time to move the ion channel data (and the rest of it) out and host the data in a separate repo. That's almost feasible now. I'm looking into what it will take to do.